src/PagesBody/ProfilePage/ProfilePage.js

```diff
@@ -1,16 +1,19 @@
 import propTypes from 'prop-types';
+import { useContext } from 'react';
 
 import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
 import FakeButton from '../../Atoms/FakeButton/FakeButton';
 import ProfileCard from '../../Molecules/ProfileCard/ProfileCard';
+import { UserDataContext } from '../../Context/UserDataContext';
 
-const ProfilePage = ({ userId }) => {
+const ProfilePage = () => {
+  const userData = useContext(UserDataContext);
   return (
     <ColumnWrapper paddingLeftRight={1}>
-      <ProfileCard />
+      <ProfileCard userData={userData.state} />
       <FakeButton to="/user-dogs" text="Twoje Psy" ternary="ternary" />
       <FakeButton
-        to={`./${userId}/contests`}
+        to={`./${userData.state.userId}/contests`}
```

Comment:
it's a place for discussion.
maybe these: 
`http://localhost:3000/user/contests` (and using the userId from UserDataContext not from useParams)
looks better than:
`http://localhost:3000/user/matylda1234/contests`

Reply:
Yes, I think the latter should be enough. No need for user's Id, the application shuold know the currently logged in user.

Reply:
Thanx for this comment. Refactored